### ISSUE TITLE
Autorotation stops after one revolution

### ISSUE THREAD

**Issue by sitting-bull** [mpetroff/pannellum#764]
Why does the autorotation stop after one revolution? Is it possible to make the pano rotate infinitely?

**Comment by mpetroff** [comment]
This was a regression introduced in 12516e69b90e8ab4c6f42b64560aef92ae539bd0. I just fixed it in d773cc9f60974bae65a38da5ead56abeb62ea963.

**Comment by auriiolym** [comment]
Here's a temporary monkeypatch fix for those that want to have a fix before the next official release:
```js
const pannellumViewer = pannellum.viewer(element, { options })

applyFix()
pannellumViewer.on('load', applyFix)

function applyFix () {
  const renderer = pannellumViewer.getRenderer()
  if (!renderer) { return }
  const next = renderer.getCanvas
  renderer.getCanvas = function (...args) {
    const config = pannellumViewer.getConfig()
    if (config.autoRotate !== false) {
      if (config.yaw > 180) {
        config.yaw -= 360
      } else if (config.yaw < -180) {
        config.yaw += 360
      }
    }
    return next(...args)
  }
}
```
